src/sagemaker/serializers.py

```diff
@@ -357,3 +357,33 @@ def serialize(self, data):
             return data.read()
 
         raise ValueError("Unable to handle input format: %s" % type(data))
+
+
+class ImageSerializer(IdentitySerializer):
+    """Serialize data in a image file by returning raw bytes rom an image file."""
+
+    def __init__(self, content_type="image/jpeg"):
+        """Initialize a ``ImageSerializer`` instance.
+
+        Args:
+            content_type (str): The MIME type to signal to the inference endpoint when sending
+                request data (default: "image/jpeg").
+        """
+        super(ImageSerializer, self).__init__(content_type=content_type)
+
+    def serialize(self, data):
+        """Serialize image of various formats to a raw bytes.
+
+        Args:
+            data (object): Data to be serialized. The data can be a string,
+                representing file-path or the raw bytes form an image.
+        Returns:
+            raw-bytes: The data serialized as a raw-bytes from the image file.
+        """
+        if isinstance(data, str):
+            image = open(data, "rb")
+            return image.read()
+        if isinstance(data, bytes):
+            return data
+
+        raise ValueError("Object of type %s is not Image serializable." % type(data))
```

Comment:
nit: can move to using fstrings in python3.

Reply:
+1. Let's use `f` strings

Reply:
updated in current revision